docs/Parameters.rst

```diff
@@ -127,7 +127,7 @@ Core Parameters
 
    -  number of boosting iterations
 
-   -  **Note**: for Python/R-package, **this parameter is ignored**, use ``num_boost_round`` (Python) or ``nrounds`` (R) input arguments of ``train`` and ``cv`` methods instead
+   -  **Note**: for R-package, **this parameter is ignored**, use ``nrounds`` input arguments of ``train`` and ``cv`` methods instead
```

Comment:
It will overwrite the default wrapper parameter when provided

Reply:
So I can remove the whole line, not only for Python, right?